### ISSUE TITLE
Account is screwed

### ISSUE THREAD

**Issue by Circlepuller** [keybase/keybase-issues#1892]
So, accidentally deleted my device keys like the dumbass I am. After trying everything I decided to try deleting my count and get re-invited. Turns out account info doesn't really get deleted as I can't use my username again. Any reasoning behind this?

**Comment by zQueal** [comment]
I have no idea if you can use the same username, but you can use the same email and get re-invited by using an email proxy. Use my public key and encrypt your email, post it here and I'll shoot you an invite. We'll test whether or not you can use the same name once your account is deleted. ;)

**Comment by Circlepuller** [comment]
Oh, I had my bro send me an invite thru another email I use. After that is how I found out I couldn't use my username even though I deleted it.

**Comment by zQueal** [comment]
I suppose it makes sense. If I were to delete my account for whatever reason, and someone else were to snipe the username while it were available then they could use it to impersonate my online presence. I really think there should be a big fat warning about accounts when they're created citing information such as;

 * You cannot delete the account and reuse the;
  * username
  * email

**Comment by Circlepuller** [comment]
The email is reusable but you have to be invited from a different email. I was just hoping someone could help me out so I could use my account name.

**Comment by oconnor663** [comment]
Could you clarify what you mean by deleting your account? I don't think we have an actual delete button.

What you can do is reset your account, which wipes out all the proofs and tracking statements you've made. It's the "reset my keys and start from scratch" option here:

![image](https://cloud.githubusercontent.com/assets/860932/11463271/6f25da98-96ed-11e5-931f-f4d62e8d4bc0.png)

Is that what you did? If so, you should be able to log in on the command line using the same password you had before, and then you can start recreating your proofs.

**Comment by Circlepuller** [comment]
If you go to the "Edit my account/settings", there's a link to delete your account.

**Comment by zQueal** [comment]
![](https://i.imgur.com/V2ls8AC.png)

**Comment by Circlepuller** [comment]
That is exactly my current situation.

**Comment by oconnor663** [comment]
Wow, joke's on me, I had no idea that was a thing.

**Comment by oconnor663** [comment]
@malgorithms is deletion unrecoverable by design?

**Comment by malgorithms** [comment]
yeah, deletion is unrecoverable by design - we don't recycle usernames because it can cause confusion / social attacks. sorry!

**Comment by Circlepuller** [comment]
Alright. Well looks like i'm using an alt.

**Comment by Circlepuller** [comment]
Thinking about it, I'm sorta relieved because nobody will be able to use my other username now >:D

**Comment by felixjendrusch** [comment]
This is still status quo, right? I ran into the "no device name reuse" issue, deleted my account because I wanted to start from scratch, and cannot reuse my account name now.

**Comment by cjb** [comment]
Yes, still status quo.

**Comment by felixjendrusch** [comment]
@cjb and there is no way, whatsoever, even with a paper key, to reclaim the previous username?

**Comment by cjb** [comment]
@felixjendrusch If the account deletion is very recent, I suggest emailing Chris and Max (@keybase.io) to ask if it can be undone.

**Comment by felixjendrusch** [comment]
@cjb thanks for your help!
